cmd/restic/cmd_check.go

```diff
@@ -399,7 +413,17 @@ func runCheck(ctx context.Context, opts CheckOptions, gopts GlobalOptions, args
 		return errors.Fatal("repository contains errors")
 	}
 
-	Verbosef("no errors were found\n")
+	if !gopts.JSON {
+		Verbosef("no errors were found\n")
+	} else {
+		status := checkSuccess{
+			MessageType: "checked",
+			Message:     "no errors were found",
+			ID:          repo.Config().ID,
```

Comment:
The `ID` and `Repository` URL aren't really relevant here and thus should be removed.